.github/workflows/R-CMD-check.yaml

```
@@ -2,9 +2,15 @@
 # Need help debugging build failures? Start at https://github.com/r-lib/actions#where-to-find-help
 on:
   push:
-    branches: [main, master]
+    branches: main
+    paths: |
+      - 'R/**'
+      - 'tests/**'
   pull_request:
-    branches: [main, master]
+    branches: main
+    paths: |
+      - 'R/**'
+      - 'tests/**'
```

Consider adding a few more paths.
"man" -- sometimes we get stray .Rd files with commits that they are not connected to. 
If there are changes to .Rd files, we should still run the tests that check the .Rd files are formatted correctly (test-return.R, test-examples.R, and test-non_ascii.R)

"vignettes" -- updates to the vignettes should still be checked with R-CMD-Check.
```suggestion
      - 'tests/**'
      - 'man/**'
      - 'vignettes/**'
```

---

Good point; I have committed the changes.